### PR TITLE
Disable Rails caching on QA for dashboard and cohort analytics

### PR DESCRIPTION
**Pull request by timmyjose**
Chore story: https://www.pivotaltracker.com/story/show/167535135
Depends on: https://github.com/simpledotorg/deployment/pull/98

This PR will avoid caching dashboard analytics in QA to avoid affecting testing by the QA team.

### CODE REVIEW

File: app/controllers/analytics/districts_controller.rb [hunk 1]
```diff
@@ -2,7 +2,11 @@ class Analytics::DistrictsController < AnalyticsController
   before_action :set_organization_district
 
   def show
-    @analytics = Rails.cache.fetch(analytics_cache_key) { analytics }
+    if ENV.fetch('SIMPLE_SERVER_ENV') == 'qa'
```
Comment (govindkrjoshi) [inline app/controllers/analytics/districts_controller.rb]:
```suggestion
    if FeatureToggle.enabled(:uncached_queries_for_dasboard)
```

File: app/controllers/analytics/facilities_controller.rb [hunk 2]
```diff
@@ -2,7 +2,11 @@ class Analytics::FacilitiesController < AnalyticsController
   before_action :set_facility
 
   def show
-    @analytics = Rails.cache.fetch(analytics_cache_key) { analytics }
+    if ENV.fetch('SIMPLE_SERVER_ENV') == 'qa'
```
Comment (govindkrjoshi) [inline app/controllers/analytics/facilities_controller.rb]: Same here.

File: app/controllers/analytics/districts_controller.rb (hunk 1 above)
Comment (timmyjose) [reply app/controllers/analytics/districts_controller.rb]: @govindkrjoshi Added new env variable `ENABLE_CACHED_QUERIES_FOR_DASHBOARD`. Toggling on the same.

File: app/controllers/analytics/facilities_controller.rb (hunk 2 above)
Comment (timmyjose) [reply app/controllers/analytics/facilities_controller.rb]: @govindkrjoshi Added new env variable ENABLE_CACHED_QUERIES_FOR_DASHBOARD. Toggling on the same.

File: .env.test [hunk 3]
```diff
@@ -13,6 +13,7 @@ ENABLE_PATIENT_FOLLOWUPS=true
 ENABLE_PHONE_NUMBER_MASKING=true
 ENABLE_API_V3=true
 ENABLE_DASHBOARD_VIEW_BY_DISTRICT=true
+ENABLE_CACHED_QUERIES_FOR_DASHBOARD=true
```
Comment (vkrmis) [inline .env.test]:
```suggestion
ENABLE_CACHED_QUERIES_FOR_DASHBOARD=false
```

**Pull request merged**